ext/oauth-client/Civi/Api4/OAuthSessionToken.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Civi\Api4;
+
+use Civi\Api4\Generic\AbstractAction;
+use Civi\Api4\Generic\Result;
+
+/**
+ * OAuth Access Tokens.
+ *
+ * @see https://docs.civicrm.org/dev/en/latest/framework/oauth/#model-token
+ *
+ * @primaryKey cardinal
+ * @searchable none
+ * @since 5.67
+ * @package Civi\Api4
+ */
+class OAuthSessionToken extends Generic\AbstractEntity {
+
+  const ENTITY = 'OAuthSessionToken';
+
+  public static function create($checkPermissions = TRUE): Generic\BasicCreateAction {
+    $action = new Generic\BasicCreateAction(
+      self::ENTITY,
```

Comment:
E.g. references to the constant could be:
```suggestion
      static::getEntityName(),
```
And the constant can be removed.

Reply:
done.